cypress/integration/inventory/update-effective-location-for-item.spec.js

```diff
@@ -30,45 +30,46 @@ describe('Update the effective location for the item', () => {
           cy.getInstanceTypes({ limit: 1 });
           cy.getLocations({ limit: 1 });
           cy.getHoldingTypes({ limit: 1 });
-          InventoryHoldings.getHoldingSources({ limit: 1 }).then(holdingsSources => {
-            sourceId = holdingsSources[0].id;
-          });
         })
           .then(() => {
-            cy.createInstance({
+            InventoryInstances.createFolioInstanceViaApi({
               instance: {
                 instanceTypeId: Cypress.env('instanceTypes')[0].id,
                 title: instanceTitle,
-                source: 'FOLIO',
               },
               holdings: [{
+                holdingsTypeId: Cypress.env('holdingsTypes')[0].id,
                 permanentLocationId: Cypress.env('locations')[0].id,
-                sourceId
               }],
               items: [
-                [{
+                {
                   barcode: ITEM_BARCODE,
                   missingPieces: '3',
                   numberOfMissingPieces: '3',
                   status: { name: 'Available' },
                   permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
                   materialType: { id: Cypress.env('materialTypes')[0].id },
                 }],
-              ],
-            });
+            })
+              .then(specialInstanceIds => {
+                testInstanceIds = specialInstanceIds;
+              });
           });
         cy.login(userProperties.username, userProperties.password);
       });
     cy.visit(TopMenu.inventoryPath);
   });
 
   afterEach(() => {
-    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${ITEM_BARCODE}"` })
-      .then((instance) => {
-        cy.deleteItem(instance.items[0].id);
-        cy.deleteHoldingRecord(instance.holdings[0].id);
-        cy.deleteInstanceApi(instance.id);
+    cy.wrap(testInstanceIds.holdingIds.forEach(holdingsId => {
+      cy.wrap(holdingsId.itemIds.forEach(itemId => {
+        cy.deleteItem(itemId);
+      })).then(() => {
+        cy.deleteHoldingRecord(holdingsId.id);
       });
+    })).then(() => {
+      cy.deleteInstanceApi(testInstanceIds.instanceId);
```

Comment:
Please rename method deleteInstanceApi to deleteInstanceViaApi

Reply:
Done